### ISSUE TITLE
Barrier not getting installed using latest dmg on MACOS 14.3 (23D56) , M2 chip

### ISSUE THREAD

**Issue by akashicMarga** [debauchee/barrier#2010]
### What happened?

I am getting below message while installing

“Barrier” cannot be opened because the developer cannot be verified. macOS cannot verify that this app is free from malware.
<img width="257" alt="Screenshot 2024-02-01 at 2 05 32 PM" src="https://github.com/debauchee/barrier/assets/18519731/3d0d447f-eebc-49d4-9815-ccc845cc1e9b">

### Version

v2.4.0

### What OSes are you seeing the problem on? (Check all that apply)

macOS

### What OS versions are you using?

MACOS 14.3 (23D56)

**Comment by Ninjapods** [comment]
Yes, it's installing, but the OS treats non App Store Apps differently now.. You have to go to System Settings > Privacy and Security, All the way to the bottom, you'll see a button asking to Allow the App to run.

**Comment by mike2212** [comment]
> Yes, it's installing, but the OS treats non App Store Apps differently now.. You have to go to System Settings > Privacy and Security, All the way to the bottom, you'll see a button asking to Allow the App to run.

Sadly does not resolve issue on a managed machine.  The app is presently unrunable, given this issue.

**Comment by asciidisco** [comment]
@mike2212 You have to decide yourself if that's in-line with your companies guidelines, but

`xattr -rd com.apple.quarantine /Applications/Barrier.ap`

should get Barrier out of quarantine, e.g. should enable you to open it.

**Comment by jswhisperer** [comment]
First you want to go to Settings > Secuirty then click this for most github downloaded apps etc

<img width="407" alt="image" src="https://github.com/debauchee/barrier/assets/1177690/955fc260-1c4a-48bd-8adb-770fcd60b0bb">

Now just right click the actually .app and click open it will show a similar prompt but allow you to click open now
